### PR TITLE
Update documentation with latest OVA changes

### PR DESCRIPTION
**Pull request by edmundoa**
OVAs in Graylog 3.0 won't use the omnibus package any more, and they also don't ship the `graylog-ctl` script. This PR takes care of reflecting those changes in the documentation.

### CODE REVIEW

File: pages/configuration/graylog_ctl.rst [hunk 1]
```diff
@@ -1,501 +0,0 @@
-.. _graylog-ctl:
```
Comment (jalogisch) [inline pages/configuration/graylog_ctl.rst:1]: can we have that page restored as `:orphan:` page - that just contains:

```
.. :orphan:

.. _graylog-ctl:
.. _extend_ova_disk:
.. _static_ip_ova:
.. _graylog_ctl_advanced:
.. _restore_etcd:

**********************
The graylog-ctl script
**********************

The script to configure the Graylog appliance is no longer available, please follow the configuration and installation instructions for Ubuntu based package installations

```

or something similar!

File: pages/configuration/file_location.rst [hunk 2]
```diff
@@ -137,65 +141,3 @@ MongoDB
 | Log files     | ``/var/log/mongodb/`` |
 +---------------+-----------------------+
 
-
-Omnibus package
-===============
-
-This paragraph covers Graylog installations via OVA, on AWS (via AMI), and on OpenStack using the `Graylog Omnibus package <https://github.com/Graylog2/omnibus-graylog2#readme>`_.
-
-.. _omnibus-graylog:
```
Comment (jalogisch) [inline pages/configuration/file_location.rst:146]: did you checked if the `:refs:` that are deleted are used somewhere?

File: pages/installation/openstack.rst [hunk 3]
```diff
@@ -1,60 +0,0 @@
-.. _OpenStack:
```
Comment (jalogisch) [inline pages/installation/openstack.rst]: same as for `pages/configuration/graylog_ctl.rst`

File: pages/installation/vagrant.rst [hunk 4]
```diff
@@ -1,41 +0,0 @@
-.. _Vagrant:
```
Comment (jalogisch) [inline pages/installation/vagrant.rst]: same as for `pages/configuration/graylog_ctl.rst`

File: pages/installation/virtual_machine_appliances.rst [hunk 5]
```diff
@@ -60,25 +60,15 @@ credentials.
 
 The web interface is reachable on port 80 at the IP address of your virtual
 machine. The login prompt of the shell is showing you this IP address, too. (See
-screenshot above). DHCP should be enabled in your network otherwise take a look into
-the ``graylog-ctl`` command to apply a static IP address to the appliance.
+screenshot above).
 
-The standard user for the web interface is *admin* with the password *admin*.
+The standard user for the web interface is *admin*, the password is shown on first boot.
```
Comment (jalogisch) [inline pages/installation/virtual_machine_appliances.rst]: .. in the console of the virtual server.

People tend to not look at that if they do not know they should.

**Review by jalogisch — APPROVED** [review]
Thank you for this @edmundoa 

I had just left a view comments - I'll approve this, as I did not know if I'm able to review the changes in time.

File: pages/configuration/file_location.rst (hunk 2 above)
Comment (edmundoa) [reply pages/configuration/file_location.rst:146]: Refs should be checked by the travis build (I saw the build fail yesterday when I did a change), but I can manually look for them 👍

**Comment by edmundoa** [comment]
Fixed all review comments and rebased the PR.

**Pull request merged**